CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 3.0.0-alpha.11
 
+### Changed
```

Comment:
Is this required for the tool to work?

Reply:
Yeah, it seems to require at least one subheading on the latest version, though it doesn't necessarily need to be that exact one.

Reply:
It pulls the first subheading?

If so what if we had a first heading `#Changelog` and the subheading becomes the version?

Reply:
@bruno-garcia The #Changelog part should be fine. It selects the version by using the regex `^## (.+)`. What I meant was that the latest version requires at least one heading within, for example:

```md
# Changelog
...

## 1.0.1

### Added:
- ...
- ...
```

Reply:
Got it. maybe a good reason to start breaking changes into ### Fixes, Feature, etc